Tabloid/Repositories/TagRepository.cs

```diff
@@ -37,6 +38,59 @@ public List<Tag> GetAllTags()
 
                 }
             }
+
+
+        }
+
+
+        public void Add(Tag tag)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @" INSERT Into Tag(Name)
+	                   OUTPUT INSERTED.ID values(@Name)";
+                    cmd.Parameters.AddWithValue("@name", tag.Name);
+
+                    tag.Id = (int)cmd.ExecuteScalar();
+                };
+
+            }
+        }
+
+
+
+
+        public Tag getTagById(int id)
```

Comment:
PascalCase naming please.

Reply:
@joshdbarton added PascalCasing to the GetTagById